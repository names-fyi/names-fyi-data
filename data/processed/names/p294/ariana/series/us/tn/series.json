[{"y":1990,"t":5,"m":0,"f":5,"mr":null,"fr":674,"mp":0,"fp":0.00016560678325384207},{"y":1992,"t":5,"m":0,"f":5,"mr":null,"fr":714,"mp":0,"fp":0.0001680728763992067},{"y":1993,"t":14,"m":0,"f":14,"mr":null,"fr":307,"mp":0,"fp":0.00047904191616766467},{"y":1994,"t":9,"m":0,"f":9,"mr":null,"fr":467,"mp":0,"fp":0.0003080398398192833},{"y":1995,"t":12,"m":0,"f":12,"mr":null,"fr":377,"mp":0,"fp":0.00040958427196395656},{"y":1996,"t":10,"m":0,"f":10,"mr":null,"fr":447,"mp":0,"fp":0.0003377009320545725},{"y":1997,"t":22,"m":0,"f":22,"mr":null,"fr":250,"mp":0,"fp":0.0007480703186099494},{"y":1998,"t":31,"m":0,"f":31,"mr":null,"fr":200,"mp":0,"fp":0.0010155610155610155},{"y":1999,"t":19,"m":0,"f":19,"mr":null,"fr":294,"mp":0,"fp":0.00062050947093403},{"y":2000,"t":21,"m":0,"f":21,"mr":null,"fr":290,"mp":0,"fp":0.0006740491092922485},{"y":2001,"t":30,"m":0,"f":30,"mr":null,"fr":204,"mp":0,"fp":0.0009944641495674082},{"y":2002,"t":35,"m":0,"f":35,"mr":null,"fr":178,"mp":0,"fp":0.0011659282454445517},{"y":2003,"t":30,"m":0,"f":30,"mr":null,"fr":209,"mp":0,"fp":0.0009869395006086128},{"y":2004,"t":32,"m":0,"f":32,"mr":null,"fr":201,"mp":0,"fp":0.0010506270930461619},{"y":2005,"t":45,"m":0,"f":45,"mr":null,"fr":146,"mp":0,"fp":0.001448342452526553},{"y":2006,"t":51,"m":0,"f":51,"mr":null,"fr":137,"mp":0,"fp":0.001605793450881612},{"y":2007,"t":54,"m":0,"f":54,"mr":null,"fr":127,"mp":0,"fp":0.0016601082144613872},{"y":2008,"t":47,"m":0,"f":47,"mr":null,"fr":152,"mp":0,"fp":0.0014613519059759965},{"y":2009,"t":40,"m":0,"f":40,"mr":null,"fr":166,"mp":0,"fp":0.0013116474291710388},{"y":2010,"t":62,"m":0,"f":62,"mr":null,"fr":106,"mp":0,"fp":0.002116980230136238},{"y":2011,"t":42,"m":0,"f":42,"mr":null,"fr":157,"mp":0,"fp":0.0014204065068145694},{"y":2012,"t":51,"m":0,"f":51,"mr":null,"fr":129,"mp":0,"fp":0.001709344416141574},{"y":2013,"t":50,"m":0,"f":50,"mr":null,"fr":139,"mp":0,"fp":0.0016844658558771014},{"y":2014,"t":84,"m":0,"f":84,"mr":null,"fr":79,"mp":0,"fp":0.0027039206849932404},{"y":2015,"t":81,"m":0,"f":81,"mr":null,"fr":81,"mp":0,"fp":0.0026050041808709074},{"y":2016,"t":84,"m":0,"f":84,"mr":null,"fr":76,"mp":0,"fp":0.0027105517909002904},{"y":2017,"t":65,"m":0,"f":65,"mr":null,"fr":98,"mp":0,"fp":0.0021298207673908056},{"y":2018,"t":62,"m":0,"f":62,"mr":null,"fr":116,"mp":0,"fp":0.0019943386515697374},{"y":2019,"t":50,"m":0,"f":50,"mr":null,"fr":135,"mp":0,"fp":0.0016397743670470944},{"y":2020,"t":41,"m":0,"f":41,"mr":null,"fr":164,"mp":0,"fp":0.0013555511472591417},{"y":2021,"t":42,"m":0,"f":42,"mr":null,"fr":169,"mp":0,"fp":0.0013513948325235689},{"y":2022,"t":38,"m":0,"f":38,"mr":null,"fr":188,"mp":0,"fp":0.0012137085183174167}]