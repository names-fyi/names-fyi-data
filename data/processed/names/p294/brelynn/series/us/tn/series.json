[{"y":2012,"t":5,"m":0,"f":5,"mr":null,"fr":1089,"mp":0,"fp":0.00016758278589623274},{"y":2022,"t":6,"m":0,"f":6,"mr":null,"fr":1034,"mp":0,"fp":0.00019163818710275}]