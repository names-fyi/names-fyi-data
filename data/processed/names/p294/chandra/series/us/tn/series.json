[{"y":1955,"t":5,"m":0,"f":5,"mr":null,"fr":562,"mp":0,"fp":0.00013009314669303222},{"y":1957,"t":10,"m":0,"f":10,"mr":null,"fr":388,"mp":0,"fp":0.00026549142462698453},{"y":1958,"t":8,"m":0,"f":8,"mr":null,"fr":441,"mp":0,"fp":0.00021830486274081756},{"y":1959,"t":8,"m":0,"f":8,"mr":null,"fr":454,"mp":0,"fp":0.0002186688533552002},{"y":1960,"t":5,"m":0,"f":5,"mr":null,"fr":601,"mp":0,"fp":0.00013735509037964946},{"y":1961,"t":11,"m":0,"f":11,"mr":null,"fr":381,"mp":0,"fp":0.00030392617356947476},{"y":1962,"t":5,"m":0,"f":5,"mr":null,"fr":582,"mp":0,"fp":0.0001404297149276787},{"y":1963,"t":11,"m":0,"f":11,"mr":null,"fr":382,"mp":0,"fp":0.00031666522727926994},{"y":1964,"t":14,"m":0,"f":14,"mr":null,"fr":324,"mp":0,"fp":0.0004018369690011481},{"y":1965,"t":9,"m":0,"f":9,"mr":null,"fr":401,"mp":0,"fp":0.00028120606155288236},{"y":1966,"t":23,"m":0,"f":23,"mr":null,"fr":224,"mp":0,"fp":0.000758225093953979},{"y":1967,"t":13,"m":0,"f":13,"mr":null,"fr":322,"mp":0,"fp":0.00043460818400641884},{"y":1968,"t":15,"m":0,"f":15,"mr":null,"fr":299,"mp":0,"fp":0.000514385652069545},{"y":1969,"t":32,"m":0,"f":32,"mr":null,"fr":195,"mp":0,"fp":0.0010528740170433982},{"y":1970,"t":28,"m":0,"f":28,"mr":null,"fr":213,"mp":0,"fp":0.0008889735530367972},{"y":1971,"t":25,"m":0,"f":25,"mr":null,"fr":231,"mp":0,"fp":0.000803548470043713},{"y":1972,"t":26,"m":0,"f":26,"mr":null,"fr":215,"mp":0,"fp":0.000918630533865668},{"y":1973,"t":19,"m":0,"f":19,"mr":null,"fr":257,"mp":0,"fp":0.0006923945920338181},{"y":1974,"t":22,"m":0,"f":22,"mr":null,"fr":233,"mp":0,"fp":0.000800378360679594},{"y":1975,"t":28,"m":0,"f":28,"mr":null,"fr":190,"mp":0,"fp":0.0010592819581583626},{"y":1976,"t":25,"m":0,"f":25,"mr":null,"fr":204,"mp":0,"fp":0.0009552558175079287},{"y":1977,"t":31,"m":0,"f":31,"mr":null,"fr":182,"mp":0,"fp":0.0011052875530359752},{"y":1978,"t":15,"m":0,"f":15,"mr":null,"fr":297,"mp":0,"fp":0.0005383097075183922},{"y":1979,"t":22,"m":0,"f":22,"mr":null,"fr":230,"mp":0,"fp":0.0007722007722007722},{"y":1980,"t":15,"m":0,"f":15,"mr":null,"fr":298,"mp":0,"fp":0.0005276116778051355},{"y":1981,"t":23,"m":0,"f":23,"mr":null,"fr":214,"mp":0,"fp":0.000824638772363845},{"y":1982,"t":13,"m":0,"f":13,"mr":null,"fr":313,"mp":0,"fp":0.00046332596763846317},{"y":1983,"t":8,"m":0,"f":8,"mr":null,"fr":427,"mp":0,"fp":0.0002900021750163126},{"y":1984,"t":10,"m":0,"f":10,"mr":null,"fr":364,"mp":0,"fp":0.0003644049267546097},{"y":1985,"t":10,"m":0,"f":10,"mr":null,"fr":372,"mp":0,"fp":0.0003600489666594657},{"y":1986,"t":9,"m":0,"f":9,"mr":null,"fr":406,"mp":0,"fp":0.0003266550522648084},{"y":1987,"t":10,"m":0,"f":10,"mr":null,"fr":388,"mp":0,"fp":0.0003543711683617421},{"y":1988,"t":8,"m":0,"f":8,"mr":null,"fr":478,"mp":0,"fp":0.00027653911300079504},{"y":1991,"t":8,"m":0,"f":8,"mr":null,"fr":505,"mp":0,"fp":0.00026304540821359287},{"y":1992,"t":7,"m":0,"f":7,"mr":null,"fr":565,"mp":0,"fp":0.00023530202695888936},{"y":1993,"t":6,"m":0,"f":6,"mr":null,"fr":625,"mp":0,"fp":0.00020530367835757058},{"y":1995,"t":9,"m":0,"f":9,"mr":null,"fr":473,"mp":0,"fp":0.00030718820397296744},{"y":1998,"t":6,"m":0,"f":6,"mr":null,"fr":735,"mp":0,"fp":0.00019656019656019656},{"y":1999,"t":6,"m":0,"f":6,"mr":null,"fr":721,"mp":0,"fp":0.00019595035924232528},{"y":2001,"t":6,"m":0,"f":6,"mr":null,"fr":753,"mp":0,"fp":0.00019889282991348163}]