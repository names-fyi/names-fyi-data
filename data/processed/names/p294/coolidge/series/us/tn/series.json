[{"y":1923,"t":5,"m":5,"f":0,"mr":447,"fr":null,"mp":0.00020667135121729425,"fp":0},{"y":1924,"t":11,"m":11,"f":0,"mr":255,"fr":null,"mp":0.00043739313690405185,"fp":0}]