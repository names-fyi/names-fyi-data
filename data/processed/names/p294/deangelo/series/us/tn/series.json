[{"y":1976,"t":7,"m":7,"f":0,"mr":367,"fr":null,"mp":0.00023508076703495986,"fp":0},{"y":1977,"t":5,"m":5,"f":0,"mr":491,"fr":null,"mp":0.00015665632734906162,"fp":0},{"y":1978,"t":6,"m":6,"f":0,"mr":435,"fr":null,"mp":0.00018785810451172548,"fp":0},{"y":1980,"t":10,"m":10,"f":0,"mr":326,"fr":null,"mp":0.0002981959147159684,"fp":0},{"y":1981,"t":5,"m":5,"f":0,"mr":457,"fr":null,"mp":0.0001544020010499336,"fp":0},{"y":1982,"t":9,"m":9,"f":0,"mr":341,"fr":null,"mp":0.0002759804973781853,"fp":0},{"y":1983,"t":12,"m":12,"f":0,"mr":280,"fr":null,"mp":0.0003768489149891656,"fp":0},{"y":1984,"t":18,"m":18,"f":0,"mr":215,"fr":null,"mp":0.0005733031818326592,"fp":0},{"y":1985,"t":15,"m":15,"f":0,"mr":242,"fr":null,"mp":0.0004652028284331969,"fp":0},{"y":1986,"t":21,"m":21,"f":0,"mr":200,"fr":null,"mp":0.000670391061452514,"fp":0},{"y":1987,"t":33,"m":33,"f":0,"mr":150,"fr":null,"mp":0.001023954325431302,"fp":0},{"y":1988,"t":24,"m":24,"f":0,"mr":184,"fr":null,"mp":0.0007091150834687547,"fp":0},{"y":1989,"t":27,"m":27,"f":0,"mr":190,"fr":null,"mp":0.0007794907327212888,"fp":0},{"y":1990,"t":26,"m":26,"f":0,"mr":199,"fr":null,"mp":0.0007297631076681262,"fp":0},{"y":1991,"t":32,"m":32,"f":0,"mr":180,"fr":null,"mp":0.0009050285649640817,"fp":0},{"y":1992,"t":31,"m":31,"f":0,"mr":186,"fr":null,"mp":0.0008815834376066431,"fp":0},{"y":1993,"t":22,"m":22,"f":0,"mr":230,"fr":null,"mp":0.0006386808337687976,"fp":0},{"y":1994,"t":26,"m":26,"f":0,"mr":211,"fr":null,"mp":0.0007514450867052023,"fp":0},{"y":1995,"t":14,"m":14,"f":0,"mr":326,"fr":null,"mp":0.00040546802594995364,"fp":0},{"y":1996,"t":30,"m":30,"f":0,"mr":195,"fr":null,"mp":0.0008726257308240495,"fp":0},{"y":1997,"t":19,"m":19,"f":0,"mr":272,"fr":null,"mp":0.0005480716531571811,"fp":0},{"y":1998,"t":11,"m":11,"f":0,"mr":407,"fr":null,"mp":0.0003052926646498848,"fp":0},{"y":1999,"t":19,"m":19,"f":0,"mr":276,"fr":null,"mp":0.0005255587519362691,"fp":0},{"y":2000,"t":8,"m":8,"f":0,"mr":532,"fr":null,"mp":0.00021898010018339583,"fp":0},{"y":2001,"t":6,"m":6,"f":0,"mr":640,"fr":null,"mp":0.00016779931202282072,"fp":0},{"y":2002,"t":12,"m":12,"f":0,"mr":400,"fr":null,"mp":0.0003433869398500544,"fp":0},{"y":2003,"t":11,"m":11,"f":0,"mr":452,"fr":null,"mp":0.0003084429240389199,"fp":0},{"y":2004,"t":10,"m":10,"f":0,"mr":507,"fr":null,"mp":0.00027848171767523464,"fp":0},{"y":2005,"t":19,"m":19,"f":0,"mr":339,"fr":null,"mp":0.0005159118062343868,"fp":0},{"y":2006,"t":10,"m":10,"f":0,"mr":529,"fr":null,"mp":0.000267079750013354,"fp":0},{"y":2007,"t":14,"m":14,"f":0,"mr":439,"fr":null,"mp":0.00036272249138534084,"fp":0},{"y":2008,"t":11,"m":11,"f":0,"mr":528,"fr":null,"mp":0.00029199405393926523,"fp":0},{"y":2009,"t":15,"m":15,"f":0,"mr":410,"fr":null,"mp":0.0004134623335814107,"fp":0},{"y":2010,"t":9,"m":9,"f":0,"mr":577,"fr":null,"mp":0.0002562861292251616,"fp":0},{"y":2011,"t":14,"m":14,"f":0,"mr":432,"fr":null,"mp":0.0004023104112187132,"fp":0},{"y":2012,"t":10,"m":10,"f":0,"mr":568,"fr":null,"mp":0.00027997088302816507,"fp":0},{"y":2013,"t":14,"m":14,"f":0,"mr":439,"fr":null,"mp":0.0003941108577541311,"fp":0},{"y":2014,"t":7,"m":7,"f":0,"mr":754,"fr":null,"mp":0.00019335948290149715,"fp":0},{"y":2015,"t":8,"m":8,"f":0,"mr":688,"fr":null,"mp":0.00021935837674801206,"fp":0},{"y":2016,"t":18,"m":18,"f":0,"mr":405,"fr":null,"mp":0.0004980630879911456,"fp":0},{"y":2017,"t":12,"m":12,"f":0,"mr":517,"fr":null,"mp":0.0003315283456735551,"fp":0},{"y":2018,"t":8,"m":8,"f":0,"mr":719,"fr":null,"mp":0.00022586747226065106,"fp":0},{"y":2019,"t":10,"m":10,"f":0,"mr":604,"fr":null,"mp":0.0002850058426197737,"fp":0},{"y":2021,"t":7,"m":7,"f":0,"mr":829,"fr":null,"mp":0.0001961718465375669,"fp":0}]