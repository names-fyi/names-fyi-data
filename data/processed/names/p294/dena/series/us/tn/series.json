[{"y":1915,"t":5,"m":0,"f":5,"mr":null,"fr":436,"mp":0,"fp":0.00023323071182013247},{"y":1919,"t":8,"m":0,"f":8,"mr":null,"fr":354,"mp":0,"fp":0.00030917874396135266},{"y":1921,"t":5,"m":0,"f":5,"mr":null,"fr":507,"mp":0,"fp":0.00017587055926837847},{"y":1945,"t":5,"m":0,"f":5,"mr":null,"fr":454,"mp":0,"fp":0.0001675603217158177},{"y":1947,"t":5,"m":0,"f":5,"mr":null,"fr":534,"mp":0,"fp":0.00012457023269719466},{"y":1950,"t":6,"m":0,"f":6,"mr":null,"fr":473,"mp":0,"fp":0.00016349664831870946},{"y":1951,"t":5,"m":0,"f":5,"mr":null,"fr":519,"mp":0,"fp":0.00013185654008438817},{"y":1952,"t":6,"m":0,"f":6,"mr":null,"fr":496,"mp":0,"fp":0.000162000162000162},{"y":1954,"t":9,"m":0,"f":9,"mr":null,"fr":404,"mp":0,"fp":0.0002324680356450988},{"y":1955,"t":14,"m":0,"f":14,"mr":null,"fr":316,"mp":0,"fp":0.0003642608107404902},{"y":1956,"t":6,"m":0,"f":6,"mr":null,"fr":525,"mp":0,"fp":0.00015626220798499883},{"y":1957,"t":10,"m":0,"f":10,"mr":null,"fr":390,"mp":0,"fp":0.00026549142462698453},{"y":1958,"t":11,"m":0,"f":11,"mr":null,"fr":368,"mp":0,"fp":0.00030016918626862415},{"y":1959,"t":9,"m":0,"f":9,"mr":null,"fr":425,"mp":0,"fp":0.0002460024600246002},{"y":1960,"t":12,"m":0,"f":12,"mr":null,"fr":357,"mp":0,"fp":0.00032965221691115873},{"y":1961,"t":21,"m":0,"f":21,"mr":null,"fr":253,"mp":0,"fp":0.00058022269499627},{"y":1962,"t":26,"m":0,"f":26,"mr":null,"fr":233,"mp":0,"fp":0.0007302345176239292},{"y":1963,"t":19,"m":0,"f":19,"mr":null,"fr":269,"mp":0,"fp":0.0005469672107551026},{"y":1964,"t":19,"m":0,"f":19,"mr":null,"fr":277,"mp":0,"fp":0.0005453501722158439},{"y":1965,"t":15,"m":0,"f":15,"mr":null,"fr":300,"mp":0,"fp":0.00046867676925480396},{"y":1966,"t":14,"m":0,"f":14,"mr":null,"fr":311,"mp":0,"fp":0.00046152831805894377},{"y":1967,"t":15,"m":0,"f":15,"mr":null,"fr":286,"mp":0,"fp":0.0005014709815458679},{"y":1968,"t":37,"m":0,"f":37,"mr":null,"fr":167,"mp":0,"fp":0.0012688179417715442},{"y":1969,"t":21,"m":0,"f":21,"mr":null,"fr":246,"mp":0,"fp":0.0006909485736847301},{"y":1970,"t":40,"m":0,"f":40,"mr":null,"fr":164,"mp":0,"fp":0.001269962218623996},{"y":1971,"t":26,"m":0,"f":26,"mr":null,"fr":226,"mp":0,"fp":0.0008356904088454615},{"y":1972,"t":24,"m":0,"f":24,"mr":null,"fr":228,"mp":0,"fp":0.000847966646645232},{"y":1973,"t":23,"m":0,"f":23,"mr":null,"fr":228,"mp":0,"fp":0.0008381618745672534},{"y":1974,"t":17,"m":0,"f":17,"mr":null,"fr":283,"mp":0,"fp":0.0006184741877978681},{"y":1975,"t":12,"m":0,"f":12,"mr":null,"fr":345,"mp":0,"fp":0.00045397798206786973},{"y":1976,"t":13,"m":0,"f":13,"mr":null,"fr":327,"mp":0,"fp":0.0004967330251041228},{"y":1977,"t":9,"m":0,"f":9,"mr":null,"fr":421,"mp":0,"fp":0.0003208899347523799},{"y":1978,"t":5,"m":0,"f":5,"mr":null,"fr":639,"mp":0,"fp":0.00017943656917279743},{"y":1979,"t":14,"m":0,"f":14,"mr":null,"fr":319,"mp":0,"fp":0.0004914004914004914},{"y":1980,"t":6,"m":0,"f":6,"mr":null,"fr":546,"mp":0,"fp":0.00021104467112205416},{"y":1981,"t":8,"m":0,"f":8,"mr":null,"fr":450,"mp":0,"fp":0.0002868308773439461},{"y":1982,"t":8,"m":0,"f":8,"mr":null,"fr":437,"mp":0,"fp":0.00028512367239290045},{"y":1983,"t":9,"m":0,"f":9,"mr":null,"fr":397,"mp":0,"fp":0.0003262524468933517},{"y":1984,"t":9,"m":0,"f":9,"mr":null,"fr":404,"mp":0,"fp":0.00032796443407914874},{"y":1985,"t":7,"m":0,"f":7,"mr":null,"fr":502,"mp":0,"fp":0.000252034276661626},{"y":1987,"t":5,"m":0,"f":5,"mr":null,"fr":654,"mp":0,"fp":0.00017718558418087105},{"y":1989,"t":5,"m":0,"f":5,"mr":null,"fr":696,"mp":0,"fp":0.0001678866429386878},{"y":1990,"t":7,"m":0,"f":7,"mr":null,"fr":555,"mp":0,"fp":0.0002318494965553789},{"y":1994,"t":6,"m":0,"f":6,"mr":null,"fr":648,"mp":0,"fp":0.00020535989321285552},{"y":1996,"t":5,"m":0,"f":5,"mr":null,"fr":765,"mp":0,"fp":0.00016885046602728624}]