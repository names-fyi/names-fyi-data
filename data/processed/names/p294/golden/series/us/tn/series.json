[{"y":1916,"t":5,"m":0,"f":5,"mr":null,"fr":476,"mp":0,"fp":0.00021640337589266391},{"y":1922,"t":7,"m":0,"f":7,"mr":null,"fr":418,"mp":0,"fp":0.00024826216484607745}]